mockery/generator.go

```diff
@@ -277,6 +278,15 @@ func (g *Generator) genList(list *ast.FieldList, addNames bool) ([]string, []str
 	return names, types, params
 }
 
+func (g *Generator) paramsAsArgNames(params []string) []string {
+	args := []string{}
+	r := regexp.MustCompile("(.+?) (\\.{3})?.+")
```

Comment:
I tried it out, and it does work, but I replaced the regex with direct use of the AST, which is how I really should have done it to begin with: https://github.com/ryanbrainard/mockery/pull/4